libs/ui/elements/src/lib/record-api-form/record-api-form.component.ts

```diff
@@ -70,4 +71,34 @@ export class RecordApiFormComponent {
     this.limit$.next(DEFAULT_PARAMS.LIMIT)
     this.format$.next(DEFAULT_PARAMS.FORMAT)
   }
+
+  parseOutputFormats() {
+    const apiUrl =
+      this.apiBaseUrl.slice(-1) === '?'
+        ? this.apiBaseUrl.slice(0, -1)
+        : this.apiBaseUrl
```

Comment:
does that cause an error down the line? having a trailing "?" should not break anything in principle

Reply:
Yes. it creates an error when `ogc-client` tries to get to the conformance document and ends up here : `https://mel.integration.apps.gs-fr-prod.camptocamp.com/data/ogcapi?%2F=&f=json`

Reply:
then I guess this is an issue with ogc-client, I'll see if I can reproduce on its side